src/NServiceBus.AcceptanceTests/Recoverability/Retries/When_custom_policy_does_single_delayed_retry_before_move_to_error.cs

```diff
@@ -36,19 +37,20 @@ public RetryEndpoint()
                 {
                     var scenarioContext = (Context) context.ScenarioContext;
                     configure.EnableFeature<TimeoutManager>();
-                    configure.SecondLevelRetries().CustomRetryPolicy(RetryPolicy);
+                    configure.Recoverability()
+                        .CustomPolicy(RetryPolicy);
                     configure.Notifications.Errors.MessageSentToErrorQueue += (sender, message) => { scenarioContext.MessageSentToErrorQueue = true; };
                 });
             }
 
-            TimeSpan RetryPolicy(SecondLevelRetryContext slrRetryContext)
+            RecoverabilityAction RetryPolicy(RecoverabilityConfig config, ErrorContext context)
             {
                 if (count == 0)
```

Comment:
Makes sense. My intention was to preserve the structure of the old tests as much as possible. But go ahead!